apiary.apib

```diff
@@ -3102,6 +3102,20 @@ Register a new webhook.
 
 + Response 204
 
+### webhooks.list [GET /webhooks.list]
+
+List registered webhooks.
+
++ Request (application/json;charset=utf-8)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + url: `https://example.org/webhook` (string) - Webhook URL
+                + types: company.added, contact.deleted (array[string]) - Event types that fire the webhook
```

Comment:
Not sure if this should be an array of an enum type here?

Reply:
But then we need to list every enum option?

Reply:
We do it already for the register endpoint anyway. Not sure if it's worth it though.